backend/src/entities/game.ts

```diff
@@ -58,6 +67,23 @@ export class Game {
   receivedAnswersForCurrentTrack: Array<User>;
   roomManager: GameRoomManager;
 
+  /**
+   * Sets the game's tracks. Suffles the tracks and add the requested
+   * number of tracks to the game. If numberSongs is not between 1 and
+   * the number of tracks, then all tracks are added to the game.
+   * It also sets the game's questions number (persisted to DB).
+   */
```

Comment:
```suggestion
```
The code speaks for itself :)

Reply:
Fair enough, but let's at least keep the precision that the tracks are in memory while the number of questions is persisted.